integration_tests/test_boost_workflow.py

```diff
@@ -0,0 +1,29 @@
+import logging
```

Comment:
Seems like you've created a new python package. Can I suggest to add `integration_tests/__init__.py`?